CONTRIBUTING.md

```diff
@@ -17,4 +17,12 @@ Atlassian requires contributors to sign a Contributor License Agreement, known a
 Prior to accepting your contributions we ask that you please follow the appropriate link below to digitally sign the CLA. The Corporate CLA is for those who are contributing as a member of an organization and the individual CLA is for those contributing as an individual.
 
 * [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
-* [CLA for individuals](https://opensource.atlassian.com/individual)
\ No newline at end of file
+* [CLA for individuals](https://opensource.atlassian.com/individual)
+
+## Code Owners' Responsibilities
+
+For any pull requests, code owners should review the changes thoroughly and make sure all the requirements are met. Code owners should also run the internal Bamboo CI tests and E2E tests against the changes.
+
+### How to E2E test
+
+The Data Center Helm Charts uses [Deployment Automation for Atlassian DC on K8s](https://github.com/atlassian-labs/data-center-terraform#deployment-automation-for-atlassian-dc-on-k8s) for end-to-end testing. Internal reviewers can run the tests by adding `e2e` label on a pull request (for external contributions, be mindful of the changes as it will run on internal cloud environment).
```

Comment:
```suggestion
The Data Center Helm Charts uses the latest release of [Deployment Automation for Atlassian DC on K8s](https://github.com/atlassian-labs/data-center-terraform#deployment-automation-for-atlassian-dc-on-k8s) for end-to-end testing. Internal reviewers can run the tests by adding `e2e` label on a pull request (for external contributions, be mindful of the changes as it will run on internal cloud environment).
```